### PR TITLE
My tutorial topic

### PR DESCRIPTION
**Pull request by radhikasundararaman24**
Merge my tutorial topic after review.

### CODE REVIEW

File: docs/tutorials/find-tasks-by-user-id.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Find tasks by user ID
+
```
Comment (rbwatson) [inline docs/tutorials/find-tasks-by-user-id.md:7]: Markdown files can only have 1 H1 heading.
Please update the other H1 headings to be H2 or below (depending on the topic).

File: docs/tutorials/find-tasks-by-user-id.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+In this tutorial, you learn the operations to call to find tasks by auto-generated user ID.
+
+Expect this tutorial to take about 15 minutes to complete.
+
+# Before you start
+
+1. Make sure you've completed the [Before-you-start-a-tutorial] topic on the development system you'll use for the tutorial.
+1. Check whether the {{base_url}} in Postman collection points to http://localhost:3000/. If not set correctly, you will see local host errors. For ****assistance setting up the variables for your import collection, refer to [set environment variables in Postman](https://learning.postman.com/docs/sending-requests/variables/environment-variables/).
```
Comment (rbwatson) [inline docs/tutorials/find-tasks-by-user-id.md]: The double brace (\{\{ and \}\}) hide the text between them when the topic is published.
See `docs/tutorials/get-a-user-by-name.md` for examples of how to annotate them to appear in the published docs.

File: docs/tutorials/find-tasks-by-user-id.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+1. (**Optional**) Make sure to select the **GET** method type. 
+1. (**Optional**) Rename the request. Here's an example: **Get tasks by user_id_2**. 
+1. Save the new request.
+1. Specify an user ID to the base URL request. For example: http://localhost:3000/tasks?user_id=1. Alternatively, you can modify the **Query Params** tab and then add a user ID. 
+1. Click **Send** to initiate your new request.  
+
+| Property  | Type | Description|
+|---|---|---|
+| User_id  | Integer   | (**Required**) Indicates a numeric value associated with each task a specific user owns.|
+
```
Comment (rbwatson) [inline docs/tutorials/find-tasks-by-user-id.md:33]: I'm not sure what the preceding table refers to. It looks like you've finished the procedure already.

File: docs/tutorials/find-tasks-by-user-id.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        "description": "Daughter's first concert appearance",
+        "due_date": "2024-04-02T15:00",
+        "warning": "-30",
+        "id": "2"
+    }
+]
+
+# Troubleshooting tips
+
+- Error handling (*link coming soon*)
```
Comment (rbwatson) [inline docs/tutorials/find-tasks-by-user-id.md]: The error handling topic has been merged.
Please add the link here.

File: docs/tutorials/find-tasks-by-user-id.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+- Error handling (*link coming soon*) 
+- Parameters for a successful request (*link coming soon*)
+
+
+# Next steps
+
+1. Integrate this service in your workflow for further downstream analysis.  
+1. Create an automated workflow between tasks and users using this service and a set of services included in the collection.
+
```
Comment (rbwatson) [inline docs/tutorials/find-tasks-by-user-id.md:72]: Good next steps!

File: docs/tutorials/find-tasks-by-user-id.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+1. Expand the **tasks** folder.
+1. Expand the **tasks:READ** method.
+1. From the list, select *Get tasks by user_id* and click on the three dots.
+1. From the options, select **Duplicate** and make a copy of the *Get tasks by user_id* request.
+1. (**Optional**) Make sure to select the **GET** method type. 
+1. (**Optional**) Rename the request. Here's an example: **Get tasks by user_id_2**. 
+1. Save the new request.
+1. Specify an user ID to the base URL request. For example: http://localhost:3000/tasks?user_id=1. Alternatively, you can modify the **Query Params** tab and then add a user ID. 
+1. Click **Send** to initiate your new request.  
+
```
Comment (rbwatson) [inline docs/tutorials/find-tasks-by-user-id.md]: Good procedure!

**Review by rbwatson — CHANGES_REQUESTED** [review]
This is a good start on your tutorial. The comments are mostly about markdown characteristics.

Thanks!

File: docs/tutorials/find-tasks-by-user-id.md (hunk 2 above)
Comment (radhikasundararaman24) [reply docs/tutorials/find-tasks-by-user-id.md]: Great tip, thx!

File: docs/tutorials/find-tasks-by-user-id.md (hunk 3 above)
Comment (radhikasundararaman24) [reply docs/tutorials/find-tasks-by-user-id.md:33]: How do we code tables between steps in Markdown? 

I wanted to keep table after step 10, so the user is aware of the accepted input type. But keeping that table after step 10 resulted step 11 to restart numbering to step 1. So I moved that table out of the procedure.

**Comment by radhikasundararaman24** [comment]
Review requested for pr-72

File: docs/tutorials/find-tasks-by-user-id.md (hunk 3 above)
Comment (rbwatson) [reply docs/tutorials/find-tasks-by-user-id.md:33]: You need to indent the table to be 1 level more indented than the procedure step you want it to be in.

**Review by rbwatson — CHANGES_REQUESTED** [review]
Be sure to update your pull request status. Right now it's a **Draft** pull request so I can't merge it.

**Comment by rbwatson** [comment]
It looks like this needs:
1. Sync fork
2. In clone, pull main branch of fork
3. In feature branch, rebase to main
4. Push feature branch to fork (this get's it up-to-date)
5. In feature branch, squash your commits
6. Push feature branch to fork (this pushes the single commit)
7. Update this pull request to be active, again (not Draft). In Draft mode, I can't do much with it.

Then it should be ready to go!

Thanks.

**Comment by radhikasundararaman24** [comment]
I followed your instructions for most part and opened a pull request again,
but am not sure how to submit a pull request with "no draft mode". What I
did differently last time was I selected "convert to draft" in my pull
request. I thought the exact opposite - that this will give more control
for you to freely edit in draft mode.

Also. I got stranded in step 4. After rebasing to main, I do not see a push
feature. I only got a message to create a Pull request. I assumed that the
pull request will update and reopen the existing pull request # 72.  How do
I do a push command from GitHub Desktop after rebasing a feature branch
with the main branch?

I'm still not getting the hang of it! Hopefully with my next practice, I
get better. Thx!

**Review by rbwatson — APPROVED** [review]
LGTM!

**Pull request merged**